python-sdk/src/astro/sql/operators/cleanup.py

```diff
@@ -158,14 +158,7 @@ def _is_dag_running(self, task_instances: list[TaskInstance]) -> bool:
             for ti in task_instances
             if ti.task_id != self.task_id
             and ti.state
-            not in [
-                State.SUCCESS,
-                State.FAILED,
-                State.SKIPPED,
-                State.UPSTREAM_FAILED,
-                State.REMOVED,
-                State.SHUTDOWN,
-            ]
+            not in [State.SUCCESS, State.FAILED, State.SKIPPED, State.UPSTREAM_FAILED, State.REMOVED]
```

Comment:
```suggestion
            not in [TaskInstanceState.SUCCESS, TaskInstanceState.FAILED, TaskInstanceState.SKIPPED, TaskInstanceState.UPSTREAM_FAILED, TaskInstanceState.REMOVED]
```
State is for Dagrun while TaskInstanceState is for Taskinstance. You need to import it though